### DISCUSSION TITLE
识别模型推理时 卡在copy_to_cpu 无法继续运行

### DISCUSSION THREAD

**Discussion by Baiy97** [PaddlePaddle/PaddleOCR#12496] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：Windows11 + wsl2 (Ubuntu22.04)
- 版本号/Version：Paddle：  paddlepaddle-gpu2.5.1.post117       PaddleOCR： v2.6 
- 问题相关组件/Related components：模型推理时 tensor.copy_to_cpu() 操作
- 运行指令/Command Code：正常跑 rec_r34_vd_none_bilstm_ctc  识别推理模型
- 完整报错/Complete Error Message：有时会卡在 tensor.copy_to_cpu() 处，不直接报错，程序也kill不掉；检查了显存内存都是没问题，这个tensor也正常（tensor.shape()可正常输出），该错误与输入图片应该无直接关系（失败的图片重跑是可以顺利通过，之前成功的图下一次可能就失败了）

其他可能有帮助的信息：
1. 同样环境下跑 det_res18_db_v2.0 检测推理模型不会出现问题（也有 copy_to_cpu 操作）
2. 识别模型推理时会根据一个batch内的最大长宽比对图片进行resize，如果对最大长宽比进行限制（比如max_wh_ratio=12）就能大大减少上述错误发生概率（我测了3000+图未出现问题，但似乎无法确定出错概率为0）

我们提供了AceIssueSolver来帮助你解答问题，你是否想要它来解答(请填写yes/no)?/We provide AceIssueSolver to solve issues, do you want it? (Please write yes/no):no

请尽量不要包含图片在问题中/Please try to not include the image in the issue.

**Comment by wwbitejotunn** [comment]
我这边试了一下好像不太可以复现该问题(连续运行一万次)，请问是否可以提供一个稳定复现的输入集。或者是否可以在纯linux环境下（而不是wsl）尝试一下是否还会有该问题发生
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/105858416/4dbc98e7-9a99-47be-a39e-592d1ef8bb0d)

**Comment by Baiy97** [comment]
> 我这边试了一下好像不太可以复现该问题(连续运行一万次)，请问是否可以提供一个稳定复现的输入集。或者是否可以在纯linux环境下（而不是wsl）尝试一下是否还会有该问题发生 ![image](https://user-images.githubusercontent.com/105858416/267320731-4dbc98e7-9a99-47be-a39e-592d1ef8bb0d.png)

纯linux环境下确实不曾出过问题，就是 Windows11 + wsl2 (Ubuntu22.04) 环境下出问题很奇怪
